.github/workflows/tag_image_push_rhel.yml

```diff
@@ -42,6 +42,10 @@ jobs:
       fail-fast: false
       matrix:
         jdk: ${{ fromJSON(needs.jdks.outputs.jdks) }}
+        variant: [ 'slim','' ]
```

Comment:
Why have you added it?

We haven't push slim versions to OpenShift Catalog

Reply:
I _assumed_ it was an oversight as it’s in the non-RHEL job. If that’s wrong I can take it out instead and update the params to suit?

Reply:
There's no request for it so I would keep it as it is (i.e. no slim version on RHEL)

Reply:
bff66f46b7260e7f3b25124010093e23ec1eec79